### PR TITLE
[15.0][MIG] sale_invoicing_date_selection: Migration to version 15.0

### PR DESCRIPTION
**Pull request by ernesto-garcia-tecnativa**
@tecnativa TT41413

### CODE REVIEW

**Comment by ernesto-garcia-tecnativa** [comment]
@pedrobaeza @victoralmau please review

File: sale_invoicing_date_selection/tests/test_sale_invoicing_date_selection.py [hunk 1]
```diff
@@ -24,6 +24,8 @@ def _create_sale_order(self, partner):
 
     def test_andvance_invoice(self):
         self.sale_order_1 = self._create_sale_order(self.partner)
+        for line in self.sale_order_1.order_line:
+            line.product_id_change()
```
Comment (pedrobaeza) [inline sale_invoicing_date_selection/tests/test_sale_invoicing_date_selection.py]: This shouldn't be needed created things with `Form`. Why do you need it?

Comment (ernesto-garcia-tecnativa) [reply sale_invoicing_date_selection/tests/test_sale_invoicing_date_selection.py]: @pedrobaeza sorry add this for a local problem, but I forgot to remove it. I pushed the change, please review

**Review by pedrobaeza — APPROVED** [review]

**Review by victoralmau — APPROVED** [review]
Code and functional review OK.

**Pull request merged**